### DISCUSSION TITLE
Home Shiny Sprites

### DISCUSSION THREAD

**Discussion by josephjoshua30** [kwsch/PKHeX#3908] [General]
Will you be updating them to Pokemon Home Shiny Sprites?

**Comment by sora10pls** [comment]
There are no plans to switch to using Pokémon renders from HOME with PKHeX. [A plugin for that already exists](https://github.com/DarkKrimson/PKHeX-Sprite-Plugin), though it requires your own dumps and they must be formatted to fit the 68x56 canvas (from the initial 512x512 resolution images).

The current SV style sprites are taken from HOME battle data, and no Shiny assets in that style exist. Like with past-gen sprites, fan made Shiny sprites will need to be made first before PKHeX can consider adding them.
